### PR TITLE
Added support for event adapters on query side.

### PR DESCRIPTION
**Pull request by aenevala**
Refs #65

### CODE REVIEW

File: src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala [hunk 1]
```diff
@@ -56,6 +59,8 @@ class JdbcReadJournal(config: Config)(implicit val system: ExtendedActorSystem)
   val db = SlickDatabase.forConfig(config, readJournalConfig.slickConfiguration)
   sys.addShutdownHook(db.close())
 
+  val eventAdapters = Persistence(system).adaptersFor(JdbcAsyncWriteJournal.Identifier)
```
Comment (aenevala) [inline src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala]: Other option is to fetch the identifier from the configuration.

**Comment by dnvriend** [comment]
Hi, thanks for the PR, I'll take a look at it

File: src/main/scala/akka/persistence/jdbc/journal/JdbcAsyncWriteJournal.scala [hunk 2]
```diff
@@ -33,6 +33,9 @@ import scala.collection.immutable._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
+object JdbcAsyncWriteJournal {
+  val Identifier = "jdbc-journal"
```
Comment (dnvriend) [inline src/main/scala/akka/persistence/jdbc/journal/JdbcAsyncWriteJournal.scala]: final val provides byte code optimization for these kinds of identifiers.  Try it out in the REPL and look at the 
resulting byte code eg: 

```
object A { val X = "Hello" }
:javap -filter -c A
```

vs

```
object A { final val X = "Hello" }
:javap -filter -c 
```

Also calls to these identifiers (final val) are inlined in the call graph eg:

```
scala> object A { final val X = "Hello" }
scala> object B { final val Z = A.X }
scala> :javap -filter -c B
```

File: src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala (hunk 1 above)
Comment (dnvriend) [inline src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala]: I'm for the configuration solution. Please use the strategy used in the akka-persistence-inmemory plugin that uses a configuration called `write-plugin` as implemented in this commit: https://github.com/dnvriend/akka-persistence-inmemory/commit/04884163cdad89860039055766084365b91aa53e#diff-ecbc52c290666a9af5901127454d2bb8R59

File: src/main/scala/akka/persistence/jdbc/journal/JdbcAsyncWriteJournal.scala (hunk 2 above)
Comment (aenevala) [inline src/main/scala/akka/persistence/jdbc/journal/JdbcAsyncWriteJournal.scala]: Cool, that I didn't know. However I'll remove the companion object as I'll switch to use configuration instead. But thanks for the information.

File: src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala (hunk 1 above)
Comment (aenevala) [inline src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala]: Ok, sounds good.

**Comment by aenevala** [comment]
I'm done with the changes. Changed events and adapters to look similar as in inmemory-journal tests. Please re-review.

**Comment by gavares** [comment]
@aenevala thanks for doing this. My team is also in need of this change.

**Comment by dnvriend** [comment]
@aenevala thanks, its looking great. will merge it shortly

**Comment by dnvriend** [comment]
@aenevala could you please do the same for the 2.5.0.0 branch. That case the two branches are in sync and have read adapters. Thanks!

**Pull request merged**